funfact/lang/_tsrex.py

```diff
@@ -130,16 +130,24 @@ def tensor(*spec, initializer=None):
         A tensor expression representing a single tensor object.
     '''
     if len(spec) == 2 and isinstance(spec[0], str) and _is_tensor(spec[1]):
-        raise NotImplementedError()
-    elif len(spec) == 1 and _is_tensor(spec[1]):
-        raise NotImplementedError()
+        symbol = spec[0]
+        initializer = spec[1]
+        size = initializer.shape
+    elif len(spec) == 1 and _is_tensor(spec[0]):
+        symbol = f'Anonymous_{AbstractTensor.n_nameless}'
```

Comment:
Need to increment `AbstractTensor.n_nameless` after using it just like a few lines down:
```
AbstractTensor.n_nameless += 1
```